pkg/controllers/virtualmachineinstance_controller.go

```diff
@@ -20,32 +20,44 @@ import (
 	"context"
 
 	"github.com/go-logr/logr"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
+
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/event"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 
 	v1 "kubevirt.io/api/core/v1"
+
+	"github.com/AlonaKaplan/kubesecondarydns/pkg/zonemgr"
 )
 
 // VirtualMachineInstanceReconciler reconciles a VirtualMachineInstance object
 type VirtualMachineInstanceReconciler struct {
 	client.Client
-	Log    logr.Logger
-	Scheme *runtime.Scheme
-	// TODO ZoneManager zoneManager
+	Log         logr.Logger
+	Scheme      *runtime.Scheme
+	ZoneManager *zonemgr.ZoneManager
 }
 
 func (r *VirtualMachineInstanceReconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.Result, error) {
 	vmi := &v1.VirtualMachineInstance{}
 	err := r.Client.Get(context.TODO(), request.NamespacedName, vmi)
 	if err != nil {
+		if apierrors.IsNotFound(err) {
+			r.Log.Info("VMI is not found, updating the zone manager the VMI no longer exists")
```

Comment:
please remove the log, it sounds it can happen a lot

Reply:
added V(3)

Reply:
We run on V(0) now ?
Do we support Vx out of the box ?
Do we even want this error ? if every VM delete gets it, it isn't an interesting one

Reply:
The default if V(0).
This is how verbosity increased - https://github.com/go-logr/zapr#increasing-verbosity.
Let's remove the log.

Reply:
yes, note that it can also be increased in the deployment itself as some repos are doing, 
without compiling

best to remove the log, thx